.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Build & Test
+
+on:
+  push:
+    branches: [master]
+  pull_request:
```

Comment:
also does this mean it'll run on every PR? or do we need some kind of wildcard here? just so i understand

Reply:
Yeah this runs on every PR. It basically says "on PR"